docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -29,20 +28,20 @@ When you promote a standby instance to active, YugabyteDB Anywhere restores your
 
 Before configuring a high-availability cluster, ensure that you have the following:
 
-* YugabyteDB Anywhere versioin 2.5.3.1 or later.
-* [Multiple YugabyteDB Anywhere instances](../../install-yugabyte-platform/) to be used in the high-availability cluster.
-* YugabyteDB Anywhere VMs can connect to each other over the port that YugabyteDB Anywhere is typically reachable (port 80 and 443, for example).
-* All YugabyteDB Anywhere instances are running the same version of YugabyteDB Anywhere software. Note that it is generally recommended to upgrade all YugabyteDB Anywhere instances in the high-availability cluster at approximately the same time.
+- YugabyteDB Anywhere version 2.5.3.1 or later.
```

Comment:
Do we need this line anymore given that 2.6 is also EOL?

Reply:
Basically all our active releases now support this feature. So why add useless info about when it was first introduced.